services/ui-src/src/components/reports/ModalDrawerReportPage.tsx

```diff
@@ -137,16 +137,17 @@ export const ModalDrawerReportPage = ({ route }: Props) => {
     verbiage.countEntitiesInTitle ? ` ${reportFieldDataEntities.length}` : ""
   }`;
 
+  const addEntityButton = (
+    <Button sx={sx.addEntityButton} onClick={addEditEntityModalOnOpenHandler}>
+      {verbiage.addEntityButtonText}
+    </Button>
+  );
```

Comment:
Request: It turns out we'll need different styling for the second button. Because of that, we might want to just un-variable-ize this one and repeat `<Button />` for each one but with different styles. The second button should have 2rem top margin and 0 bottom margin.

Reply:
ahhh okay